### ISSUE TITLE
Calling a variable named callable with one arg breaks after 2to3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5327]
|[<img alt="pitrou's avatar" src="https://avatars.githubusercontent.com/u/1721820?s=50" width="50" height="50">](https://github.com/pitrou)| @pitrou reported|
|-|-|
|Trac ID|trac#5327|
|Type|enhancement|
|Created|2011-10-21 17:43:09Z|
|Branch|https://github.com/twisted/twisted/tree/rename-callable-5327|

2to3 replaces the following code:

```
    callable(value)
```

with

```
    isinstance(value, collections.Callable)
```

This of course breaks the code when ``callable`` is a local variable you do intend to call, not the built-in function of the same name.

There are two places in Twisted where this occurs, patch attached.

Attachments:

* [callable_2to3.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/dc4/dc40b19d71eb00a80ad7bcf2b31df01b46d186a4/814a2dffa029d44f45f1947ba49ec24b841ce2eb.patch) (944 bytes) - added by antoine on 2011-10-21 17:43:17Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5327 5327
type__enhancement enhancement
reporter__antoine antoine
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_rename_callable_5327 branches/rename-callable-5327
branch_author__antoine antoine
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1319218989000000 1319218989000000
changetime__1319253260000000 1319253260000000
version__None None
owner__antoine antoine
cc__antoine cc__teratorn@...
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)|@teratorn set owner to @teratorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_3"></a>|@teratorn commented|
|-|-|

(In [32952]) Branching to 'rename-callable-5327'

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_4"></a>|@teratorn set owner to @pitrou|
|-|-|

Thanks, this looks good. I've simply renamed the 'callable' argument to 'func' in the 'fire' method. +1 merge

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_5"></a>|@teratorn set status to `closed`|
|-|-|

(In [32956]) Merge rename-callable-5327: rename variables named 'callable' in situations where it would confuse 2to3
Author: Antoine Pitrou
Reviewer: teratorn
Fixes: [#5327](https://github.com/twisted/twisted/issues/5327)
